client/src/components/ProjectUsers.vue

```diff
@@ -0,0 +1,286 @@
+    selectedPermissionSet: {},
+  }),
+  computed: {
+    ...mapState(['currentProject', 'allUsers']),
+    permissions() {
+      return this.currentProject.settings.permissions;
+    },
+    writePermission() {
+      return this.user.is_superuser;
+    },
+    members() {
+      const members = [...this.currentProject.settings.permissions.tier_1_reviewer];
+      members.push(...this.currentProject.settings.permissions.tier_2_reviewer);
+      return members;
+    },
+    collaborators() {
+      return this.currentProject.settings.permissions.collaborator;
+    },
+    changesMade() {
+      return JSON.stringify(this.permissions)
+        !== JSON.stringify(this.selectedPermissionSet);
+    },
+  },
+  watch: {
+    currentProject(newProj) {
+      this.selectedPermissionSet = { ...newProj.settings.permissions };
+    },
+  },
+  mounted() {
+    this.$store.dispatch('loadAllUsers');
+    this.selectedPermissionSet = { ...this.permissions };
+  },
+  methods: {
+    ...mapActions(['loadAllUsers']),
+    ...mapMutations(['setCurrentProject']),
+    getGroup(user) {
+      return Object.entries(this.permissions).filter(
+        ([, value]) => value.includes(user),
+      )[0][0].replace(/_/g, ' ');
+    },
+    async savePermissions() {
+      const newSettings = { ...this.currentProject.settings };
+      newSettings.permissions = Object.fromEntries(
+        Object.entries(this.selectedPermissionSet).map(
+          ([group, list]) => [group, list.map((user) => user.username || user)],
+        ),
+      );
+      try {
+        const resp = await djangoRest.setSettings(this.currentProject.id, newSettings);
+        this.showAddMemberOverlay = false;
+        this.showAddCollaboratorOverlay = false;
+        const changedProject = { ...this.currentProject };
+        changedProject.settings = resp;
+        this.setCurrentProject(changedProject);
+      } catch (e) {
+        this.$snackbar({
+          text: 'Failed to save permissions.',
+          timeout: 6000,
+        });
+      }
+    },
+  },
+};
+</script>
+
+<template>
+  <v-container class="pl-8">
+    <v-row
+      no-gutters
+      class="pb-3"
+    >
+      <v-col cols="12">
+        Members
+      </v-col>
+    </v-row>
+    <v-row
+      v-for="(user, index) in members"
+      :key="'member_'+index"
+      no-gutters
+      class="py-1"
+    >
+      <v-col cols="1">
+        <UserAvatar :targetUser="user" />
+      </v-col>
+      <v-col cols="11">
+        {{ user.username }}
+        <span class="gray-info">
+          {{ getGroup(user) }}
+        </span>
+      </v-col>
+    </v-row>
+    <v-row v-if="writePermission">
+      <v-col cols="12">
+        <v-tooltip bottom>
+          <template v-slot:activator="{ on, attrs }">
+            <v-icon
+              v-bind="attrs"
+              v-on="on"
+              @click="showAddMemberOverlay = true"
+              color="blue darken-2"
+            >
+              mdi-dots-horizontal
```

Comment:
I think something like this looks slightly cleaner than wedging a new row in:
![image](https://user-images.githubusercontent.com/577946/144114452-8a0c4851-45bc-4c8c-b1a6-438df0828f9d.png)